internal/web_package/BUILD.bazel

```diff
@@ -0,0 +1,27 @@
+package(default_visibility = ["//visibility:public"])
+
+exports_files(["web_package.bzl", "inject_html.bzl"])
+
+load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
+
+filegroup(
+    name = "node_modules_none",
+)
+
+nodejs_binary(
+    name = "assembler",
+    entry_point = "build_bazel_rules_nodejs/internal/web_package/assembler.js",
+    data = ["assembler.js"],
+    node_modules = ":node_modules_none",
+    install_source_map_support = False,
+)
+
+nodejs_binary(
+    name = "injector",
+    entry_point = "build_bazel_rules_nodejs/internal/web_package/injector.js",
+    data = [
+      "injector.js",
+      "@build_bazel_rules_nodejs_web_package_deps//parse5"
```

Comment:
Side-note: It's much easier to author new rules by using enclosed deps but adds an extra yarn install for every rule downstream that does this which is around 15s per rule since yarn_install are all run in serial with the cache mutex. Should come up with a pattern that we can follow easily so that new rules will end up with transitive deps in the user's node_modules?

Reply:
yes, we should do that for all the extra yarn_install's in this repo, which means publishing a bunch of new `@bazel/*` packages. Let's do that as a follow-up.

Reply:
👍